src/components/People.tsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import { Link, useLocation, useSearchParams } from 'react-router-dom';
+import cn from 'classnames';
+
+import { usePeopleState } from '../store/PeopleContext';
+
+import { Person } from '../types';
+
+export const People: React.FC = () => {
+  const { people } = usePeopleState();
+
+  const { pathname } = useLocation();
+
+  const [searchParams] = useSearchParams();
+  const sex = searchParams.get('sex') || "";
+  const q = searchParams.get('q') || "";
+  const centuries = Number(searchParams.get('centuries') || 0);
+
+  const getSlugByName = (name: string) => {
+    const human = people.find((person: Person) => person.name === name);
+
+    return human?.slug || null;
+  };
+
+  const setNameColor = (sex: string) => {
```

Comment:
<img width="628" alt="image" src="https://github.com/mate-academy/react_people-table-advanced/assets/70220667/e1f69b21-1971-47d5-a57c-dd29ef31fa07">
just rename the argument in this function to fix the error